### PR TITLE
Publish Legend Position.

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo, @jbrundage Please Review

File: widgets/src/c3/Common.js [hunk 1]
```diff
@@ -74,8 +79,11 @@
             height: this.height()
         };
         this._config.data.type = this._type;
+        this._config.legend = {
```
Comment (mlzummo) [inline widgets/src/c3/Common.js]: should be be moved to .update()

**Comment by mlzummo** [comment]
this._config = { ... in Common.js constructor with ... "legend: 'bottom'" should match the publish option default or might even be redundant?

File: widgets/src/c3/Common.js (hunk 1 above)
Comment (GordonSmith) [inline widgets/src/c3/Common.js]: You can't move the legend once its been rendered...

**Comment by GordonSmith** [comment]
Fixed the default value as suggested (and commented on the other issue) - please re-review...

**Comment by mlzummo** [comment]
Looks good

**Pull request merged**